### PR TITLE
Drop ruby193 references from spec files

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by pcreech** [comment]
rubygem-passenger changelog linting failed.

Also looks like the job timed out.

File: packages/foreman/tfm/tfm.spec [hunk 1]
```diff
@@ -52,7 +52,6 @@ Requires: scl-utils
 Requires: %{scl_prefix_ruby}runtime
 Requires: %{_root_bindir}/scl_source
 Requires(post): policycoreutils-python
-Obsoletes: ruby193-ruby-wrapper
```
Comment (evgeni) [inline packages/foreman/tfm/tfm.spec]: missing revision and changelog bump

**Comment by evgeni** [comment]
eventmachine (and probably others) fail to build with:

```
+ cd rubygem-eventmachine-1.0.6
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ scl enable tfm -
ERROR:  While executing gem ... (ArgumentError)
    "e>\n    <" is not an octal string
error: Bad exit status from /var/tmp/rpm-tmp.sJEro3 (%prep)
    Bad exit status from /var/tmp/rpm-tmp.sJEro3 (%prep)
```

This does not seem to be related to changes in this PR, tho?

@pcreech anything you did in the last days?

**Comment by ehelms** [comment]
[test rpm]

**Pull request closed without merging**